### PR TITLE
HasUpgradeCommenced into BDD style - OSD-5228

### PR DESCRIPTION
**Pull request by himanshudogra**
### What type of PR is this?
_test_

### What this PR does / why we need it?
In order to implement the BDD style for MUO

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-5228
_Fixes #_

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Ran `make generate` command locally to validate code changes
- [ ] Included documentation changes with PR

### CODE REVIEW

File: pkg/osd_cluster_upgrader/upgrader_verification_test.go [hunk 1]
```diff
@@ -69,12 +68,12 @@ var _ = Describe("ClusterUpgrader verification and health tests", func() {
...
-				DesiredUpdate: &configv1.Update{Version: upgradeConfig.Spec.Desired.Version + "different"},
-				Channel:       upgradeConfig.Spec.Desired.Channel,
-			},
-		}
+		/*		upgradeHasNotCommencedCV = &configv1.ClusterVersion{
+				Spec: configv1.ClusterVersionSpec{
+					DesiredUpdate: &configv1.Update{Version: upgradeConfig.Spec.Desired.Version + "different"},
+					Channel:       upgradeConfig.Spec.Desired.Channel,
+				},
+			}*/
```
Comment (mrbarge) [inline pkg/osd_cluster_upgrader/upgrader_verification_test.go:76]: No need to keep this - just delete.
```suggestion
```

Comment (himanshudogra) [reply pkg/osd_cluster_upgrader/upgrader_verification_test.go]: Removed.

**Comment by sedroche** [comment]
Missed the merge but lgtm @himanshudogra :+1:

**Pull request merged**